src/main/java/com/mcmiddleearth/mcmechat/util/NameHistoryUtil.java

```diff
@@ -0,0 +1,159 @@
+package me.gilan.mcmetesting.utils;
+
+import com.mcmiddleearth.pluginutil.message.MessageType;
+import com.mcmiddleearth.pluginutil.message.MessageUtil;
+import com.mcmiddleearth.pluginutil.NumericUtil;
+import me.gilan.mcmetesting.MCMEtesting;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandException;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import com.mcmiddleearth.pluginutil.message.FancyMessage;
+import org.bukkit.entity.Player;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class NameHistoryUtil{
+
+     public static List<FancyMessage> getFancyList(String name) {
+        //Get UUID
+        URL url = null;
+        try {
+            url = new URL("https://api.mojang.com/users/profiles/minecraft/" + name);
+        } catch (MalformedURLException e) {
+            e.printStackTrace();

```

Comment:
Use Logger.getLogger(ChatPlugin.getInstance().getClass().getSimpleName()).log(Level.WARNING,[your_message or null], e) instead of printStackTrace